src/Windows/ProfilesDialog.cpp

```diff
@@ -592,9 +612,16 @@ INT_PTR ProfilesDialog::OnInitDialog() {
 	}
 
 	if (m_profiles->size() > 0) {
+		size_t index = 0;
+		for (size_t i = 0; i < m_profiles->size(); i++) {

```

Comment:
Would it be possible to use find instead?

Reply:
Haven't thought about this, but seems to be the better way. Changed it accordingly.